app/services/pipeline_report_service.rb

```diff
@@ -95,6 +95,7 @@ def initialize(pipeline_run, background_id, csv: false, min_contig_reads: DEFAUL
     @pipeline_run = pipeline_run
     @background_id = background_id
     @csv = csv
+    raise 'bad min_contig_reads' unless min_contig_reads
```

Comment:
I use `assert` commonly when I code in python... not sure the convention in ruby, or what we what to follow

Reply:
Fine to raise an exception, but I prefer to define a custom error whenever possible so you can catch later on.
However, in this case, you might want to consider just enforcing `DEFAULT_MIN_CONTIG_READS`, with `@min_contig_reads = min_contig_reads || DEFAULT_MIN_CONTIG_READS`.

In any case, make sure the code outside properly handles the exception if you raise (my preference is just to make it work by enforcing the default value).

Reply:
> However, in this case, you might want to consider just enforcing DEFAULT_MIN_CONTIG_READS

Good idea. It's too late to try to do things "properly" . Better to be safe with existing code.